tests/e2e/encrypted.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+#
+# Copyright Confidential Containers Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(dirname "$(readlink -f "$0")")"
+source "${script_dir}/lib.sh"
+
+export key_file="${key_file:-image_key}"
+export key_path="${key_path:-/default/image_key/nginx}"
+export trustee_repo="${trustee_repo:-https://github.com/confidential-containers/trustee.git}"
+export kbs_namespace="${kbs_namespace:-coco-tenant}"
+export aa_kbc="${aa_kbc:-cc_kbc}"
+export kbs_svc_name="${kbs_svc_name:-kbs}"
+export kbs_ingress_name="${kbs_ingress_name:-kbs}"
+export runtimeclass="${runtimeclass:-kata-qemu}"
```

Comment:
Hi @GabyCT !

I started reviewing and, more importantly, running this script. I'm executing function by function, as of today I ran the `build_encryption_key()`.

The first thing caught my attention was the default value of `runtimeclass`. It should be `kata-qemu-coco-dev` and the encrypted test should not run on `kata-qemu` or `kata-clh`.

Reply:
@wainersm the `kata-qemu-coco-dev` does not exists as we can see https://github.com/confidential-containers/operator/blob/cb92e0f48ebdb9b4869ae88b9f22a669dc63eb32/.github/workflows/ccruntime_e2e.yaml#L22

Reply:
yes, not yet. It's in my TODO list to create a new job for kata-qemu-coco-dev.